src/Symfony/Component/Console/Command/Command.php

```diff
@@ -72,6 +72,15 @@ public static function getDefaultDescription(): ?string
         return null;
     }
 
+    public static function getDefaultHelp(): ?string
```

Comment:
Do we really need to make this method static?

Reply:
Not really, it's not necessary for laziness and ppl can override `getHelp()` directly anyway.

Reply:
Removed from the proposal